pkg/asset/installconfig/azure/session.go

```diff
@@ -89,7 +89,8 @@ func credentialsFromFileOrUser(cloudEnv *azureenv.Environment) (*Credentials, er
 	os.Setenv(azureAuthEnv, authFilePath)
 	_, err := auth.NewAuthorizerFromFileWithResource(cloudEnv.ResourceManagerEndpoint)
 	if err != nil {
-		logrus.Debug("Could not get an azure authorizer from file. Asking user to provide authentication info")
+		logrus.Debugf("Could not get an azure authorizer from file: %s", err.Error())
```

Comment:
Could we make this an `Infof` so it's clear to users not running with debug logs that the certificate auth failed?

Reply:
If we are going to default to asking info from the user then keeping this in debug makes sense. I see the uses of making this Infof too